flink-runtime/src/main/java/org/apache/flink/runtime/rpc/RpcEndpoint.java

```diff
@@ -137,18 +153,26 @@ public final void start() {
 	 * @throws Exception indicating that the rpc endpoint could not be started. If an exception occurs,
 	 * then the rpc endpoint will automatically terminate.
```

Comment:
The two paragraphs and the `@throws` clause should go to the `onStart` method because that's what users will override.